cognite/seismic/protos/v1/seismic_service_datatypes.proto

```diff
@@ -26,8 +26,8 @@ message LineBasedVolume {
 }
 
 message LineRange {
-    google.protobuf.Int32Value from_line = 1;
-    google.protobuf.Int32Value to_line = 2;
+    LineDescriptor inline = 1;
```

Comment:
@skairunner These used in the SDK yet?

Reply:
(That is, the previous version of these fields)

Reply:
They have never been filled in (so never have been not null), so I think it's safe to overwrite.

Reply:
:+1: